src/base/server_type.cpp

```diff
@@ -158,6 +159,7 @@ int init_server_type (const char *db_name)
     {
       // page server needs a log prior receiver
       log_Gl.initialize_log_prior_receiver ();
+      log_Gl.initialize_log_prior_sender ();
```

Comment:
it would be better to have the sender be initialized before the receiver; conceptually - although in practice it might not make a difference as it depends when is a sink actually registered with the sender - it is better to have the sender already be available when the receiver starts receiving & dispatching